riscv/isa_parser.cc

```diff
@@ -34,9 +34,9 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
   const char* all_subsets = "mafdqchpv";
 
   max_isa = reg_t(2) << 62;
-  // enable zicntr and zihpm unconditionally for backward compatibility
+  // enable zicntr and zihpm unconditionally for backward compatibility (??)
```

Comment:
As this is a specific Ibex compatible branch I'd alter this comment to something like:

enable zicntr unconditionally, zihpm is disabled as Ibex does not implement it.